ui/src/postcard.tsx

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Model, S } from 'h2o-wave'
+import React from 'react'
+import { stylesheet } from 'typestyle'
+import { Component, XComponents } from './form'
+import { cards } from './layout'
+import { XPersona } from './persona'
+import { clas, cssVar, px } from './theme'
+import { Command } from './toolbar'
+import { bond } from './ui'
+
+const css = stylesheet({
+  card: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-end',
+    padding: 24,
+    paddingBottom: 16,
+    marginTop: -4,
+    $nest: {
+      '.ms-Persona-primaryText': {
+        fontSize: 16,
+        lineHeight: px(22),
+        fontWeight: 500,
+        color: cssVar('$neutralPrimary')
+      },
+      '.ms-Persona-secondaryText': {
+        fontSize: 14,
+        lineHeight: px(20),
+        color: cssVar('$text7')
+      },
+      '.ms-Persona-image, .ms-Persona-imageArea': {
+        height: '40px !important',
+        width: '40px !important',
+      }
+    }
+  },
+  aux_value: {
+    marginBottom: 4
```

Comment:
Super nit: This misses one px.

![image](https://user-images.githubusercontent.com/64769322/141955186-f68d5dc2-a779-4186-9a19-6cae12f53cc9.png)

vs

![image](https://user-images.githubusercontent.com/64769322/141955345-e95a5ba2-a49b-407a-b9f4-db4763939f11.png)
```suggestion
    marginBottom: 5
```

I use https://chrome.google.com/webstore/detail/pixelzoomer/fogkjckfkdcnmnnfmbieljpkmmihhpao?hl=en for these things.